HelpAssistHer/help-assist-her
# Style feedback page

## Description
Add the styling for the Feedback page.

<img width="938" alt="Screen Shot 2019-04-11 at 3 50 30 PM" src="https://user-images.githubusercontent.com/6378435/55998067-94432900-5c71-11e9-8f2e-f204b6b11ee2.png">

## Test Plan
1. Go to http://localhost:4000/feedback
1. Make sure the styles match the designs
1. Make sure the styles look good on both desktop and mobile views
